src/test/java/jenkins/plugins/mailer/tasks/MimeMessageBuilderTest.java

```diff
@@ -164,4 +172,28 @@ public void test_addRecipients_tokenizer() throws Exception {
         Assert.assertEquals("tom.yyyy@gmail.com", recipients[1].toString());
         Assert.assertEquals("tom.zzzz@gmail.com", recipients[2].toString());
```

Comment:
You could use constants everywhere since you are doing it in other places, just to be consistent.